web/modules/custom/dpl_opening_hours/src/Plugin/rest/resource/v1/OpeningHoursResource.php

```diff
@@ -37,8 +37,8 @@ public function getPluginDefinition(): array {
           Request::METHOD_GET => [
             'branch_id' => [
               'name' => 'branch_id',
-              'type' => 'integer',
-              'description' => 'The id of the branch for which to retrieve opening hours.',
+              'type' => 'string',
+              'description' => 'The id(s) of the branch(es) for which to retrieve opening hours. Can be a single id or a comma-separated list of ids.',
```

Comment:
:+1: for updating the description.

You have misunderstood which resource needs to be updated.

This updates the standard resource for the opening hours at `/api/v1/opening_hours` but the actual API that needs to be updated is the legacy API used by the Redia application. This resides in [`OpeningHoursResource.php`](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/54690dd61703eea71ebeb397afdf2f3573fc3d5d/web/modules/custom/dpl_redia_legacy/src/Plugin/rest/resource/OpeningHoursResource.php#L1-L1) at `/opening_hours/instances`.